src/pages/Landing/components/StudentTicket/Top/Top.module.scss

```diff
@@ -0,0 +1,48 @@
+@import 'sass/variables';
+@import 'sass/variables';
+.TopSection{
+    width: 100%;
+    height: 13.5rem;
+    margin: 1.2rem .2rem 2.4rem 0.1rem;
+    // margin: 1rem;
+    // padding: 22px 789px 28px 49px;
+    border: solid 1px #c698f3;
+    background-color: #e6e0f2;
+    .head{
```

Comment:
Sub classes also apply upper camel case for consistency

Reply:
got it